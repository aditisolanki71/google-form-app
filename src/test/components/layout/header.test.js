import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../../components/layouts/header'

test('Snapshot of Header Component', () => {
   const component = renderer.create(
     <Header>header</Header>,
   );
   let tree = component.toJSON();
   expect(tree).toMatchSnapshot();

 });
