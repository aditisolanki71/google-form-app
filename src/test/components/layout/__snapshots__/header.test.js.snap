// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Header Component 1`] = `
<div>
  <div
    className="header"
  >
    <span
      className="total-response-text"
    >
      Total Submitted Response is : 
      0
    </span>
    <span
      className="total-response-text"
    >
      Total Response is : 
      0
    </span>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Add Form
      </span>
    </button>
  </div>
  <div
    className="table-container"
  >
    <div
      className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <table
        aria-label="customized table"
        className="MuiTable-root makeStyles-table-1"
        role={null}
      >
        <thead
          className="MuiTableHead-root"
          role={null}
        >
          <tr
            className="MuiTableRow-root MuiTableRow-head"
            role={null}
          >
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-2"
              scope="col"
            >
              UUid
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-2"
              scope="col"
            >
              Name
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-2"
              scope="col"
            >
              Description
            </th>
          </tr>
        </thead>
        <tbody
          className="MuiTableBody-root"
          role={null}
        />
      </table>
    </div>
  </div>
</div>
`;
